src/components/InputError/index.tsx

```diff
@@ -1,21 +1,13 @@
 import * as React from 'react';
 import styled, { themed } from '../../utils/styled';
-import { StandardProps } from '../../common';
-import { distance } from '../../distance';
 import { getFontStyle } from '../../textStyles';
+import { PaddedContainer, PaddedContainerProps } from '../PaddedContainer';
 
-export interface InputErrorProps extends StandardProps {
-  /**
-   * The content of input error (error to display).
-   */
-  children?: React.ReactNode;
-}
+export interface InputErrorProps extends PaddedContainerProps {}
 
-const StyledError = styled('div')<InputErrorProps>`
-  ${getFontStyle({ size: 'xSmall' })}
+const StyledError = styled(PaddedContainer)<InputErrorProps>`
```

Comment:
`styled.div`

I think there was a misunderstanding. Gutters should be up to the consumer. Semantically error/info notification style has nothing to do with `PaddedContainer`, while  `PaddedContainer` is a container and Info/Error Input is styling of error notifications. they serve different purposes. It has no sense that this component has the container props.

When you need to add gutters to InputNotification you will use composition either through styled-components api or just as a basic composition

i.e
```
// through styled-components
const NotificationWithGutter = styled(InputNotification)`
    padding: ${distance.small};
`
```
or
```
// standard composition
<PaddedContainer><InputNotification {....} /></PaddedContainer>
```